components/tricentis_qtest/common/utils.mjs

```
@@ -0,0 +1,54 @@
+export async function getFieldProps() {
+  if (this.useFields === false) return {};
+
+  const fields = await this.getDataFields();
+
+  function getFieldType(type) {
+    switch (type) {
+    case "LongText":
```

Biome

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)